### PR TITLE
Implemented Lasso and Ridge Regularization 

### PR DESCRIPTION
**Pull request by YashitaBansal**

### CODE REVIEW

File: src/slowmokit/methods/linear_model/lasso_regularization.cpp [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+{
+  this->lambda = lambda;
+}
+
+template<class T>
+double LassoRegularization<T>::lossFunction(std::vector<std::vector<T>> &x,
+                                            std::vector<T> &y)
+{
+  int n = x.size();
+  double L = 0.0;
```
Comment (ken1000minus7) [inline src/slowmokit/methods/linear_model/lasso_regularization.cpp:19]: Write a more descriptive variable name and use camel case

File: src/slowmokit/methods/linear_model/lasso_regularization.cpp [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  for (int i = 0; i < n; i++)
+  {
+    double y_pred = 0.0;
```
Comment (ken1000minus7) [inline src/slowmokit/methods/linear_model/lasso_regularization.cpp:22]: camel case as always

File: src/slowmokit/methods/linear_model/ridge_regularization.cpp [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  //   std::vector<T> yPred;
+  //   int trainExampleSize = x.size(), featureSize = x[0].size();
+  //   for (int example = 0; example < trainExampleSize; example++)
+  //   {
+  //     T currentY = coefficients[0];
+  //     for (int feature = 0; feature < featureSize; feature++)
+  //       currentY += coefficients[feature + 1] * x[example][feature];
+  //     yPred.push_back(currentY);
+  //   }
+  //   return yPred;
```
Comment (ken1000minus7) [inline src/slowmokit/methods/linear_model/ridge_regularization.cpp:99]: Delete these lines if not required, do not keep commented code anywhere

File: src/slowmokit/methods/linear_model/ridge_regularization.cpp [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  double y_pred = coefficients[0];
```
Comment (ken1000minus7) [inline src/slowmokit/methods/linear_model/ridge_regularization.cpp:100]: camel case as always

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]

**Review by Ishwarendra — CHANGES_REQUESTED** [review]

**Comment by ken1000minus7** [comment]
Write `Loss` in camel case i.e. `loss`

**Review by ken1000minus7 — APPROVED** [review]
Lgtm

**Pull request closed without merging**